build.zig.zon

```diff
@@ -1,6 +1,13 @@
 .{
-    .name = "zfat",
-    .version = "0.0.0",
+    .name = .zfat,
+    .fingerprint = 0x4212c7b5f54ad348,
+    .version = "0.15.0",
+    .dependencies = .{
+        .fatfs = .{
+            .url = "http://elm-chan.org/fsw/ff/arc/ff15a.zip",
+            .hash = "N-V-__8AAFQITQCnpmdR7PARImvk-cgb-9lZmjKolexSWkUL",
```

Comment:
I would keep vendoring the sources, as my plan is to gradually rewrite zfat fully in Zig to make the library less pain to use

Reply:
Is it even possible to rewrite it piece by piece?